backends/README.md

```diff
@@ -144,7 +144,7 @@ from your Netdata):
    When multiple servers are defined, Netdata will try the next one when the first one fails. This allows you to
    load-balance different servers: give your backend servers in different order on each Netdata.
 
-   Netdata also ships `nc-backend.sh`, a script that can be used as a fallback backend to save the
+   Netdata also ships `../exporting/nc-backend.sh`, a script that can be used as a fallback backend to save the
```

Comment:
I think we can't remove it from backends right now. Let's left backends part untouched. We will remove the hole subsystem after the transition period has ended.

Reply:
Ok I copied the file back and I am keeping the duplication for while.